word_movers_distance/Calculate_WMD_cos_scores.ipynb

```diff
@@ -0,0 +1,3817 @@
+       "4   0.678937                    cultural                    cultural  \n",
+       "5   0.682766                    cultural                    cultural  \n",
+       "6   0.470944                    cultural                  relational  \n",
+       "7   0.557336                    cultural                  relational  \n",
+       "8   0.596235                  relational                  relational  \n",
+       "9   0.627851                    cultural                  relational  \n",
+       "10  0.638289                    cultural                  relational  \n",
+       "11  0.611774                    cultural                 demographic  \n",
+       "12  0.619399                    cultural                  relational  \n",
+       "13  0.659807                    cultural                  relational  \n",
+       "14  0.601292                    cultural                  relational  \n",
+       "15  0.597985                  relational                  relational  \n",
+       "16  0.555859                    cultural                  relational  \n",
+       "17  0.502839                    cultural                  relational  \n",
+       "18  0.599960                    cultural                    cultural  \n",
+       "19  0.595760                    cultural                  relational  \n",
+       "\n",
+       "[20 rows x 23 columns]"
+      ]
+     },
+     "execution_count": 73,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "pos_demog"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 77,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "drop_columns= ['Unnamed: 0','Unnamed: 0.1','text','mean_wmd_score', 'top10_sims_articles','text_no_tags','processed']"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 82,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pos_cult = pos_cult.drop(drop_columns,axis = 1 )\n",
+    "neg_cult = neg_cult.drop(drop_columns,axis = 1 )\n",
+    "pos_demog = pos_demog.drop(drop_columns,axis = 1 )\n",
+    "neg_demog = neg_demog.drop(drop_columns,axis = 1 )\n",
+    "pos_rela = pos_rela.drop(drop_columns,axis = 1 )\n",
+    "neg_rela = neg_rela.drop(drop_columns,axis = 1 )\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 84,
+   "metadata": {
+    "id": "K8SjK1HfiTdR"
+   },
+   "outputs": [],
+   "source": [
+    "pos_cult.to_csv('pos_cult_sample_wmd_cos.csv')\n",
+    "neg_cult.to_csv('neg_cult_sample_wmd_cos.csv')\n",
+    "pos_demog.to_csv('pos_demog_sample_wmd_cos.csv')\n",
+    "neg_demog.to_csv('neg_demog_sample_wmd_cos.csv')\n",
+    "pos_rela.to_csv('pos_rela_sample_wmd_cos.csv')\n",
+    "neg_rela.to_csv('neg_rela_sample_wmd_cos.csv')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "7vb2LBlrm6wZ"
+   },
+   "source": [
+    "## Inspecting Distribution of Similarity Metrics"
```

Comment:
Nice job creating histograms here! To improve, please compare *within* metrics: we want to compare WMD scores for positive vs. negative cultural articles (based on hand-labels), not WMD scores with cosine scores for just positive cultural articles--because believe it or not, the latter doesn't tell us anything precise about the relative performance of these metrics. Comparing within metrics will help us determine which metric does a better job differentiating positive from negative labels.

Reply:
This has been implemented